content/articles/record-notes.md

```diff
@@ -24,13 +24,22 @@ When creating, updating, or deleting a DNS record from the [DNS record editor](/
 
 ## Creating a record note
 
-When you create or update a DNS record, you can fill in the _Notes_ field with the purpose of the record.
+1. Click the relevant domain in your Domain List.
+1. Click the DNS badge in the top right.
+
+![screenshot: dns badge navigation item](/files/dns-badge.png)
+
+1. To add a note to an existing record, click the **Add Note** icon on the right side, and enter your note.
```

Comment:
@dperrymorrow can you fix the numbering here and on 36? it's not showing up correctly in the preview